--- ocrad-0.14/page_image_layout.cc.orig	2006-01-02 19:31:18.000000000 +0100
+++ ocrad-0.14/page_image_layout.cc	2006-02-19 20:13:54.160338000 +0100
@@ -27,10 +27,6 @@
 #include "page_image.h"
 
 
-namespace {
-
-void find_rows( const Rectangle & rin, bool recursive ) throw();
-
 void Page_image::find_columns( const Rectangle & rin, bool recursive ) throw()
   {
   if( !this->includes( rin ) ) return;
@@ -172,9 +168,6 @@
     }
   }
 
-} // end namespace
-
-
 // Creates a reduced Page_image
 //
 Page_image::Page_image( const Page_image & source, const int scale ) throw()
